1.Hello-World/Hello-World/Hello-World-Info.plist

```diff
@@ -45,5 +45,9 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>NSCameraUsageDescription</key>
+	<string>Opentok needs access to the camera to stream your video</string>
+	<key>NSMicrophoneUsageDescription</key>
+	<string>Opentok needs access to the camera to stream your audio</string>
```

Comment:
@robjperez isn't  "Opentok needs access to the "microphone" to stream your audio ?